pt/news/_posts/2018-12-25-ruby-2-6-0-released.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: news_post
+title: "Ruby 2.6.0 Released"
+author: "naruse"
+translator: "vnbrs (Vinicius Brasil)"
+date: 2018-12-25 00:00:00 +0000
+lang: pt
+---
+
+Nós temos o prazer de anunciar o lançamento do Ruby 2.6.0.
+
+Ele introduz múltiplas novas funcionalidades e melhorias de performance, sendo as mais notáveis:
+
+ * Um novo compilador JIT.
+ * O módulo `RubyVM::AbstractSyntaxTree`.
+
+## JIT [Experimental]
+
+O Ruby 2.6 introduz uma implementação inicial de um compilador JIT (Just-In-Time).
+
+O compilador JIT foca em melhorar a performance das aplicações Ruby. Diferente dos compiladores JIT tradicionais, que operam no processo, o compilador JIT do Ruby escreve código C no disco e invoca um compilador comum C para gerar código nativo. Para mais detalhes sobre isso, acesse a [organização MJIT por Vladimir Makarov](https://bugs.ruby-lang.org/projects/ruby/wiki/MJIT#MJIT-organization).
+
+Para habilitar o compilador JIT, especifique `--jit` na linha de comando ou na variável de ambiente `$RUBYOPT`. Especificando `--jit-verbose=1` irá fazer com o que compilador JIT imprima informações adicionais. Leia o output de `ruby --help` ou [a documentação](https://bugs.ruby-lang.org/projects/ruby/wiki/MJIT#Basic-usage) para outras opções.
+
+O compilador JIT é compatível quando o Ruby é compilado pelo GCC, Clang, ou Microsoft VC++, que precisa estar disponível em tempo de execução.
+
+Com o Ruby 2.6.0, conseguimos atingir [1.7x faster performance](https://gist.github.com/k0kubun/d7f54d96f8e501bbbc78b927640f4208) se comparado ao Ruby 2.5 em um teste de benchmark de CPU não-trivial, chamado [Optcarrot](https://github.com/mame/optcarrot). De qualquer forma, ainda está em fase experimental e muitas outras aplicações de memória intensiva, como o Rails, podem não se beneficiar no momento. Para mais detalhes, veja [Ruby 2.6 JIT - Progresso e Futuro](https://medium.com/@k0kubun/ruby-2-6-jit-progress-and-future-84e0a830ecbf).
+
+Fique ligado na nova era de performance do Ruby.
+
+## `RubyVM::AbstractSyntaxTree` [Experimental]
+
+O Ruby 2.6 introduz o módulo`RubyVM::AbstractSyntaxTree`. **A compatibilidade futura desse módulo não é garantida.**.
+
+Esse módulo possui um método de `parse`, o qual traduz um dada string para código Ruby e retorna os nodos AST (Abstract Syntax Tree) desse código. O método `parse_file` abre e traduz um dado arquivo em código Ruby e retorna os nodos AST.
+
+A classe `RubyVM::AbstractSyntaxTree::Node` também foi adicionada. Você pode obter a localização do código-fonte e os nodos filhos de objetos `Node`. Essa funcionalidade é experimental.
+
+## Outras funcionalidades notáveis
+
+* Adiciona um alias de `Kernel#yield_self` chamado `#then`. [[Feature #14594]](https://bugs.ruby-lang.org/issues/14594)
```

Comment:
Use single square brackets for links.

Reply:
@jcserracampos no, it‘s ok like that, see `en`; the inner pair of brackets is part of the link text, `[Feature ...]`.